code/modules/defenses/sentry_computer.dm

```diff
@@ -0,0 +1,439 @@
+#define DEFAULT_MAP_SIZE 15
```

Comment:
shouldn't this be a var, const, or define somewhere more important? Unless I'm misunderstanding what this is